requirements/base.in

```diff
@@ -3,6 +3,10 @@
 celery>=3.0,<4.0          # Asynchronous tasks
 edx-celeryutils           # Custom task extensions
 Django                    # Web application framework
```

Comment:
I think library code should remain as flexible as it can.  We are tested through django 2.0, so no reason to cap the top end.  That just adds more work to the edx-platform upgrade.

I could add a `>=1.11`, though.

Reply:
Up to you -- does it make sense to include these other supported django versions in your .travis.yml too then?